src/Cake/Scripting/Roslyn/RoslynScriptEngine.cs

```diff
@@ -9,13 +9,13 @@ namespace Cake.Scripting.Roslyn
 {
     internal sealed class RoslynScriptEngine : IScriptEngine
     {
-        private readonly RoslynScriptSessionFactory _stableFactory;
-        private readonly RoslynNightlyScriptSessionFactory _nightlyFactory;
+        private readonly RoslynSessionFactory _stableFactory;
+        private readonly RoslynNightlySessionFactory _nightlyFactory;
```

Comment:
@patriksvensson hmm, maybe I should keep **Script** in the new name. **RoslynNightlySession** sounds ratherh kinky :smile: